src/npcmove.cpp

```diff
@@ -4489,8 +4491,23 @@ bool npc::complain()
     //Bleeding every 5 minutes
     if( has_effect( effect_bleed ) ) {
         const bodypart_id &bp = convert_bp( bp_affected( *this, effect_bleed ) );
-        std::string speech = string_format( _( "My %s is bleeding!" ), body_part_name( bp ) );
-        if( complain_about( bleed_string, 5_minutes, speech ) ) {
+        std::string speech;
+        time_duration often;
+        if( get_effect( effect_bleed, bp->token ).get_intensity() < 10 ) {
+            speech = string_format( _( "My %s is bleeding!" ), body_part_name( bp ) );
+            often = 5_minutes;
+        } else {
+            speech = string_format( _( "My %s is bleeding awfully!" ), body_part_name( bp ) );
```

Comment:
```suggestion
            speech = string_format( _( "My %s is bleeding awfully bad!" ), body_part_name( bp ) );
```
or "bad" also appropriate.

Reply:
Your suggested fix doesn't sound right to me - I'd do
```suggestion
            speech = string_format( _( "My %s is bleeding badly!" ), body_part_name( bp ) );
```
if it were to change, but I'm not sure it does.